manage.py

```diff
@@ -36,6 +37,12 @@ def campaignviews():
     views.helper.reset_campaign_views()
 
 
+@periodic.command
+def send_job_alerts():
```

Comment:
Make this `send_jobpost_alerts` since `job` and `jobpost` are distinct concepts.